routing/car_model.cpp

```diff
@@ -42,15 +44,14 @@ namespace routing
 CarModel::CarModel()
   : VehicleModel(classif(), s_carLimits)
 {
-  initializer_list<char const *> arr[] =
-  {
-    { "route", "ferry", "motorcar" },
-    { "route", "ferry", "motor_vehicle" },
-    { "railway", "rail", "motor_vehicle" },
-    { "route", "shuttle_train"},
+  vector<AdditionalRoadTags> additionalTags = {
+      {{"route", "ferry", "motorcar"}, 15.0},
```

Comment:
Please, create named constants, like in the pedestrian model or in the bicycle mode.